zrml/swaps/src/weights.rs

```diff
@@ -71,7 +72,16 @@ impl<T: frame_system::Config> WeightInfoZeitgeist for WeightInfo<T> {
     // Storage: MarketCommons Markets (r:1 w:0)
     // Storage: MarketCommons MarketPool (r:1 w:0)
     // Storage: Swaps Pools (r:1 w:1)
-    fn admin_clean_up_pool() -> Weight {
+    fn admin_clean_up_pool_cpmm_categorical(_: u32) -> Weight {
+        (87_620_000 as Weight)
+            .saturating_add(T::DbWeight::get().reads(4 as Weight))
+            .saturating_add(T::DbWeight::get().writes(2 as Weight))
+    }
```

Comment:
I wonder why the weight doesn't depend on the component for the number of assets at the moment. 🤨 It should.

Reply:
Oh, I never ran the benchmarks. I just took the one that was already there and added an used parameter to keep the PR clean.

Reply:
When I run the benchmark locally, I get

```
========
Storage: unknown [0x3a7472616e73616374696f6e5f6c6576656c3a] (r:1 w:1)
Storage: MarketCommons Markets (r:1 w:0)
Storage: MarketCommons MarketPool (r:1 w:0)
Storage: Swaps Pools (r:1 w:1)

Median Slopes Analysis
========
-- Extrinsic Time --

Model:
Time ~=    57.62
    + a    0.875
              µs

Reads = 4 + (0 * a)
Writes = 2 + (0 * a)
```

Reply:
Ok, then it looks like expected 👍